vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    image_pil.thumbnail((1024, 1024))
</pre><p>Two comments:</p><p> </p><ol><li>Shouldn't this go further below? The next section is on "Generate with Imagen", which doesn't seem to require the output of this cell. Perhaps it would be more useful if it were closer to the cells that show image editing.</li><li>I think it would be better to show a sample image from a public URL, rather than one that users upload. Open to ideas here though, but just want to remove any frictions from running this notebook by the user.</li><li>Does it make sense to force an image into a 1:1 aspect ratio? What if I upload a 16:9 image? It may be better to detect the width &amp; length, and set the larger one to be 1024 while maintaining the aspect ratio?</li></ol><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
1. I think it makes the most sense to group the base image cells together, since the user only has to use one input method.
2. For URL images: yes, we can support this in another change. 
3. The method is a little misleading here, the image is not resized in both dimensions. It only resizes the largest dimension to 1024 and scales the other dimensions proportionally, maintaining a constant aspect ratio. I edited the comment for clarity. Reference: https://pillow.readthedocs.io/en/latest/reference/Image.html#PIL.Image.Image.thumbnail